e2e-network/k8s/test-01-simple-k8s.sh

```diff
@@ -20,11 +20,7 @@ dumpLogs() {
 
 networkDown() {
   rm -rf "$TEST_LOGS"
-  (cd "$TEST_TMP" && "$FABLO_HOME/fablo-target/fabric-k8s.sh" down)
-}
-
-chainCode() {
-  (cd "$TEST_TMP" && "$FABLO_HOME/fablo-target/fabric-k8s.sh" chaincodes)
+  (cd "$TEST_TMP" && "$(find . -type f -iname 'fabric-k8s.sh')" down)
```

Comment:
why this way? I think it would be better to use file path

Reply:
I believe (cd "$TEST_TMP" && "$FABLO_HOME/fablo.sh" down) should work. And we should support it this way